### ISSUE TITLE
call service failure

### ISSUE THREAD

**Issue by Kirono** [custom-components/zaptec#28]
Call service does not work for me I get the error message "Failed to call service zaptec.start_charging. Error rendering data template: Result is not a Dictionary. Could it be because I have the wrong charger id? I could not find any id with the format: yyyyyyyy-9999-99xx-xxx-yyyyyyyyyyyy. Where can I find this id?

**Comment by thols11** [comment]
I have somewhat the same issue. I get an error message saying "required key not provided @ data['charger_id']".

**Comment by Kirono** [comment]
Yes, i get the same error when I don’t have anything in the service data field, but when I add something I get my error

**Comment by thols11** [comment]
@Kirono  what field and in whitch file have you added the Zap id?

**Comment by Kirono** [comment]
![zaptec](https://user-images.githubusercontent.com/33318089/216430463-6604378c-4800-45f4-91a5-66068ce65c55.PNG)
Just tested it in developer tools call service and there is a data field there

**Comment by thols11** [comment]
Well, I gave up, I found an integration to the car itself (Kia) on github and uses that to start/stop charging. I can also start/ stop preheating of the car.

**Comment by jorgel272** [comment]
I stumbled upon this issue and was also not able to use the Zaptec HAC integration. I had the same error message "required key not provided @ data['charger_id']".

The solution was to get the correct value from the device. I used the value  "ID" shown as an attribute of the charger.

![image](https://github.com/custom-components/zaptec/assets/50120389/c31e2b98-ae48-4708-a966-34141413be69)

Hope this works for others!
